gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagActionReminderScheduler.java

```diff
@@ -65,16 +76,20 @@ public DagActionReminderScheduler(StdSchedulerFactory schedulerFactory)
    * @param reminderDurationMillis
    * @throws SchedulerException
    */
-  public void scheduleReminder(DagActionStore.DagActionLeaseObject dagActionLeaseObject, long reminderDurationMillis)
+  public void scheduleReminder(DagActionStore.DagActionLeaseObject dagActionLeaseObject, long reminderDurationMillis,
+      boolean isDeadlineReminder)
       throws SchedulerException {
-    JobDetail jobDetail = createReminderJobDetail(dagActionLeaseObject);
+    JobDetail jobDetail = createReminderJobDetail(dagActionLeaseObject, isDeadlineReminder);
     Trigger trigger = createReminderJobTrigger(dagActionLeaseObject.getDagAction(), reminderDurationMillis,
-        System::currentTimeMillis);
+        System::currentTimeMillis, isDeadlineReminder);
+    log.info("Reminder set for dagAction {} to fire after {} ms, isDeadlineTrigger: {}",
```

Comment:
it might be more helpful to make this log show the time we expect it to fire (from the calculated date time)